include/libtorrent/settings_pack.hpp

```diff
@@ -794,7 +794,11 @@ namespace libtorrent
 
 			// time to wait until a new retry of a web seed takes place
 			urlseed_wait_retry,
-
+			
+			// time to wait until a new retry of a web seed name lookup. Default
```

Comment:
the default value will be extracted from the code by the documentation generation tool. It's better not to mention it in the comment.